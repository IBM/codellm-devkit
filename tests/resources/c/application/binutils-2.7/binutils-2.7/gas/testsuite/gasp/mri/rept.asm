	rept	3
	dc.l	1
	endr
