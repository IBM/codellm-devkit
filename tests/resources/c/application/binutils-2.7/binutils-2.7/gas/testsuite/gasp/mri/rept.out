;	rept	3
;	dc.l	1
;	endr
;	dc.l	1
	dc.l	1
;	REPT	2
;	dc.l	1
;	ENDR
;	dc.l	1
	dc.l	1
;	REPT	1
;	dc.l	1
;	ENDR
;	dc.l	1
	dc.l	1
;